src/06-extensions.md

```diff
@@ -26,41 +25,44 @@ The BIDS specification can be extended in a backwards compatible way and will
-| [BEP019](https://docs.google.com/document/d/1FqJI791ycXr0bfRg2qyLqAf0RpVttJ2cInOgMWrKsNU) | [DICOM Metadata](https://docs.google.com/document/d/1FqJI791ycXr0bfRg2qyLqAf0RpVttJ2cInOgMWrKsNU)                                                                                             | Satrajit Ghosh                                                 |
-| [BEP020](https://docs.google.com/document/d/1eggzTCzSHG3AEKhtnEDbcdk-2avXN6I94X8aUPEBVsw) | [Eye Tracking including Gaze Position and Pupil Size(ET)](https://docs.google.com/document/d/1eggzTCzSHG3AEKhtnEDbcdk-2avXN6I94X8aUPEBVsw)                                                    | Benjamin Gagl and Dejan Draschkow                              |
-| [BEP021](https://docs.google.com/document/d/1PmcVs7vg7Th-cGC-UrX8rAhKUHIzOI-uIOh69_mvdlw) | [Common Electrophysiological Derivatives](https://docs.google.com/document/d/1PmcVs7vg7Th-cGC-UrX8rAhKUHIzOI-uIOh69_mvdlw)                                                                    | Stefan Appelhoff, Cyril Pernet, Robert Oostenveld, Teon Brooks |
-| [BEP022](https://docs.google.com/document/d/1pWCb02YNv5W-UZZja24fZrdXLm4X7knXMiZI7E2z7mY) | [Magnetic Resonance Spectroscopy (MRS)](https://docs.google.com/document/d/1pWCb02YNv5W-UZZja24fZrdXLm4X7knXMiZI7E2z7mY)                                                                      | Dickson Wong                                                   |
-| [BEP023](https://docs.google.com/document/d/1yzsd1J9GT-aA0DWhdlgNr5LCu6_gvbjLyfvYq2FuxlY) | [PET Preprocessing derivatives](https://docs.google.com/document/d/1yzsd1J9GT-aA0DWhdlgNr5LCu6_gvbjLyfvYq2FuxlY)                                                                              | Martin Noergaard, Graham Searle, Melanie Ganz                  |
-| [BEP024](https://docs.google.com/document/d/1fqnJZ18x5LJC8jiJ8yvPHUGFzNBZ6gW2kywYrUKWtuo) | [Computed Tomography scan (CT)](https://docs.google.com/document/d/1fqnJZ18x5LJC8jiJ8yvPHUGFzNBZ6gW2kywYrUKWtuo)                                                                              | Hugo Boniface                                                  |
-| [BEP025](https://docs.google.com/document/d/1chZv7vAPE-ebPDxMktfI9i1OkLNR2FELIfpVYsaZPr4) | [Medical Population Imaging Data structure (MIDS)](https://docs.google.com/document/d/1chZv7vAPE-ebPDxMktfI9i1OkLNR2FELIfpVYsaZPr4)                                                           | Jose Manuel Saborit Torres                                     |
-| [BEP026](https://docs.google.com/document/d/14KC1d5-Lx-7ZSMtwS7pVAAvz-2WR_uoo5FvsNirzqJw) | [Microelectrode Recordings (MER)](https://docs.google.com/document/d/14KC1d5-Lx-7ZSMtwS7pVAAvz-2WR_uoo5FvsNirzqJw)                                                                         	  | Greydon Gilmore                                            	   |
```

Comment:
This may have some unintended conflicts with the derivatives merge due to the mismatch of this file. I don't want to add more challenges to that PR. Perhaps after that is merged this can be merged? Thank you for putting together this PR nonetheless!

Reply:
I see what you mean - thanks: I wasn't aware that there are lots of changes in the derivatives PR 

https://github.com/bids-standard/bids-specification/pull/109/files#diff-2a8145214cd54ad2715d4543399df795

I guess @chrisfilo can make the call whether we should wait with this or we can go on with it, because he'd be the one who'd have to rebase the derivatives PR.

Reply:
I would not recommend any PR to wait for #109. I am syncing it with master on a regular basis and will deal with any conflicts.

Reply:
Given @chrisfilo's [comment](https://github.com/bids-standard/bids-specification/pull/186#discussion_r268215528), could you review/approve once more @franklin-feingold ?

also: @jasmainak perhaps you could give this a quick glance. It's nothing special, but I am adding your name as a moderator of BEP021, so this is your chance to protest or approve :-)

or protestingly approve

Reply:
you know the perils of asking me to review :-) do we have renewed interest in BEP021? so far I have just been replying to people ... maybe we can look at it once the iEEG/EEG publications go through!

Reply:
I think we do have interest in BEP021, but I also think we should gather some more experience with EEG and iEEG before moving on with 100% force ;-)

re: your review of this PR: Could you make use of the GitHub tool and approve (or disapprove)? We have community guidlines (and GH settings) here that require that as a formality.

Reply:
@jasmainak